extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -528,6 +534,43 @@ export class AmpStoryPage extends AMP.BaseElement {
     ]);
   }
 
+  /* @override */
+  onLayoutMeasure() {
+    const layoutBox = this.getLayoutSize();
+    if (
+      !isPrerenderActivePage(this.element) ||
+      (this.layoutBox_ &&
+        this.layoutBox_.width === layoutBox.width &&
+        this.layoutBox_.height === layoutBox.height)
+    ) {
+      return;
+    }
+    this.getVsync().runPromise({
+      measure: (state) => {
+        const {height, width} = layoutBox;
+        state.height = height;
+        state.width = width;
+        state.vh = height / 100;
```

Comment:
This doesn't read from the DOM and doesn't need to be in a `mutate`, + you're already getting the `layoutBox` a few lines above